src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -160,18 +173,66 @@
         >Return to this step to finish it</router-link>
       </section>
 
-      <section v-if="getCreateResolutionStep.validationDetail.valid"
-        class="section-container upload-success-message"
+      <section id="resolution-summary-section-3"
+               v-if="isTypeCoop && !allResolutionValidationItemsInvalid"
+               class="section-container rounded-bl-0"
+               :class="{ 'invalid-section': !getCreateResolutionStep.validationDetail.valid }"
       >
-        <v-row no-gutters>
-          <v-col md="1">
-            <v-icon class="success-chk">mdi-check</v-icon>
+        <span v-if="!getCreateResolutionStep.validationDetail.valid">
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text ml-1 mr-2">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.CREATE_RESOLUTION}` }"
+          >Return to this step to finish it</router-link>
+        </span>
+        <v-row class="ml-2 mt-6" no-gutters>
+          <v-col md="3">
+            <strong>Resolution Date</strong>
           </v-col>
-          <v-col v-if="isTypeCoop" md="11" id="file-name-col">
-            <span>FILE_NAME_PLACEHOLDER</span>
+          <v-col md="9" class="ml-n1">
+            {{resolutionDate}}
           </v-col>
-          <v-col v-if="isBaseCompany" md="11" id="file-name-col">
-            <span>The resolution was completed and deposited in the Company's records book.</span>
+        </v-row>
+        <v-row class="ml-2 mt-5" no-gutters>
+          <v-col md="3">
+            <strong>Resolution Text</strong>
+          </v-col>
+          <v-col md="9" class="ml-n1">
+            <v-lazy>
+              <v-textarea
+                id="resolution-text"
+                rows="1"
+                auto-grow
+                readonly
+                filled
+                background-color="white"
+                v-model="resolutionText">
+              </v-textarea>
+            </v-lazy>
+          </v-col>
+        </v-row>
+        <v-row class="ml-2 mt-3" no-gutters>
+          <v-col md="3">
+            <strong>Signing Party</strong>
+          </v-col>
+          <v-col md="9" class="ml-n1">
+            {{signingParty}}
+          </v-col>
+        </v-row>
+        <v-row class="ml-2 mt-5" no-gutters>
```

Comment:
This block of code is large enough, and uses enough getters, that I would prefer to see it in its own sub-component -- that would keep THIS component (the review page) cleaner.

Doesn't have to be done in this PR if you're crunched for time.

Reply:
ok, sounds good.  I'll see how it goes.  Might be in a separate PR.